lib/rspec/matchers/built_in/change.rb

```diff
@@ -371,6 +371,14 @@ def evaluate_value_proc
           end
         end
 
+        def inspect_message_with_receiver(message, receiver)
+          if receiver.class == Class
```

Comment:
Moreover, we should check if it's `Module` or not rather than `Class` so that we can handle both classes and modules 💁